src/fromager/external_commands.py

```diff
@@ -20,7 +22,12 @@ def run(
     env = os.environ.copy()
     env.update(extra_environ)
 
-    logger.debug(
+    if network_isolation:
+        # prevent network access by creating a new network namespace that
+        # has no routing configured.
+        cmd = ["unshare", "--net", "--map-current-user"] + cmd
+
+    logger.info(
```

Comment:
Logging these commands at INFO is going to make the console output pretty verbose. Was this for your own debugging, or do you want to keep it this way?

Reply:
I didn't meant to commit the change. I bumped the log level locally for testing.